docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -268,6 +268,135 @@ spec:
 ----
 ====
 
+You could run into a situation where there are multiple configs maps that have the same property names. For example:
```

Comment:
documentation that tries to explain what are my ideas of implementing this functionality